packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -2,6 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(team-framework): Remove this tag once this test's state leaks/test
+// dependencies have been fixed.
+// https://github.com/flutter/flutter/issues/85160
+// Fails with "flutter test --test-randomize-ordering-seed=20240201"
+@Tags(<String>['no-shuffle'])
```

Comment:
Please revert this. The test isn't failing. (Well, it fails ones and then is retried to verify the retry logic, what you saw in the log is the expected first failure. Overall the test passes with the seed.)

Reply:
Done.